README.md

```diff
@@ -50,6 +50,12 @@ c.pushMetric('multiObjectDelete', '3d534b1511e5630e68f0', {
 });
 ```
 
+If an error occurs during and the client is unable to record metrics in the
+underlying datastore, metric data is instead stored in a local cache. Utapi
+attempts to push these cached metrics every five minutes, by default. The
```

Comment:
what happens if the push at a 5 minute interval fails and continues to fail?  Is there any alert action taken?